readme.md

```diff
@@ -21,6 +22,12 @@ Table of Contents
 * [Development](#development)
 * [Testing](#testing)
 
+
+Requirements
+=============
+
+* [Arcanist](https://github.com/phacility/arcanist) is installed on the Jenkins nodes where the tests will be run (e.g. not just the master) an the `arc` binary is in `$PATH` or configured explicitly in the global settings.
```

Comment:
typo: **and** the ...

Also we should note that this is only required if you use the "Apply Phabricator Differential" part of the plugin. Some people just use the post-build action (e.g. to report data to harbormaster, which can be used either with staging areas or if you're running builds on diffusion commits). But we can clarify that later. For now just fix the typo then :shipit:

Thanks!